src/nextjournal/clerk/viewer.cljc

```diff
@@ -617,13 +785,8 @@
                   (instance? clojure.lang.Namespace scope)
                   (var? scope)))
       (swap! !viewers assoc scope viewers)
-      (with-viewer :eval! `'(v/set-viewers! ~(datafy-scope scope) ~viewers)))))
-
```

Comment:
Here I'm dropping registration in the client and removing `:eval!` viewer. `set-viewers!` returns the collection it's been called with.